mobile-app/app/screens/AppNavigator/screens/Portfolio/LOCK/LockKycScreen.tsx

```diff
@@ -0,0 +1,118 @@
+          </View>
+
+          <Text style={tailwind('self-center text-xs')}>
+            {translate('LOCK/LockKycScreen', 'An additional KYC Process can be avoided by transferring your KYC data from DFX SA to MN IT Services LLC (Entity operating LOCK Staking) to start staking at LOCK immediately.\nFor this, I hereby agree to the transfer of my personal data that I have provided to DFX SA, incorporated in Switzerland, during the onboarding process to MN IT Services LLC on St.Vincent and the Grenadines for the purposes of carrying out know your customer (KYC) and anti-money-laundering (AML) procedures.I acknowledge that my personal data will be transferred to a country outside of the EU/Switzerland that is handling  data protection differently which means that my rights might not be enforceable in such country.You can withdraw your consent at any time.Please note however, that the withdrawal of your consent does not affect the lawfulness of processing based on your consent before its withdrawal.For further information please see our Privacy Notice or contact us: info@lock.space')}
+          </Text>
+        </View>
+
+        <TouchableOpacity disabled style={tailwind('bg-white rounded-md py-2 px-8 my-8')}>
+          <Text style={tailwind('ml-6 text-lg font-bold', true/* !acknowledged */ ? 'text-gray-300' : '')}>
+            {translate('LOCK/LockKycScreen', 'New KYC Process - tbd.')}
+          </Text>
+          <Text style={tailwind('ml-6 text-base', true/* !acknowledged */ ? 'text-gray-300' : '')}>
+            {translate('LOCK/LockKycScreen', 'New KYC (Know-your-customer) verification via LOCK.')}
+          </Text>
+        </TouchableOpacity>
+
+        <TouchableOpacity style={tailwind('flex-row mb-2')} onPress={async () => await Linking.openURL('https://lock.space/terms')}>
+          <MaterialCommunityIcons
+            style={tailwind('mr-2 text-lock-800 self-center')}
+            iconType='MaterialCommunityIcons'
+            name='information-outline'
+            size={24}
+          />
+          <Text style={tailwind('text-xs font-medium underline mr-8')}>
+            {translate('LOCK/LockKycScreen', 'I take note that by pressing “SUBMIT KYC DATA” I agree with the terms and conditions of MN IT Services LLC. ')}
+          </Text>
+        </TouchableOpacity>
+
+        <Button
+          fill='fill'
+          label={translate('LOCK/LockKycScreen', 'SUBMIT KYC DATA')}
+          margin=''
+          onPress={submit}
+          lock
+          isSubmitting={isSubmitting}
+          disabled={!acknowledged}
+        />
+        {/* <SubmitButtonGroup
+          isDisabled={!formState.isValid}
+          label={translate('LOCK/LockDashboardScreen', 'CONTINUE')}
+          // processingLabel={translate('components/Button', 'CONTINUE')}
+          onSubmit={onSubmit}
+          title='sell_continue'
+          isProcessing={isSubmitting}
+          displayCancelBtn={false}
+          lock
+        /> */}
```

Comment:
Will this button be used for new KYC?

Reply:
nah, can be removed

Reply:
new kyc method is already in the screen, just disabled